src/cmd/src/standalone.rs

```diff
@@ -482,6 +496,12 @@ impl StartCommand {
         .await
         .context(StartFrontendSnafu)?;
 
+        // flow server need to be able to use frontend to write insert requests back
+        flownode
+            .set_frontend_invoker(Box::new(frontend.clone()))
+            .await;
+        let _handle = flownode.clone().run_background();
```

Comment:
Consider adding `start` and `stop` to flownode to unify startup and graceful shutdown in `Instance::start` and `Instance::stop`

Reply:
> Consider adding `start` and `stop` to flownode to unify startup and graceful shutdown in `Instance::start` and `Instance::stop`

Added `TODO`